res_partner_operating_unit/security/res_partner_security.xml

```diff
@@ -0,0 +1,20 @@
+<odoo noupdate="1">
+
+    <!-- Copyright 2017 Niaga Solution - Edi Santoso <repodevs@gmail.com>
+    Copyright (C) 2019 Serpent Consulting Services
+    License LGPL-3.0 or later (http://www.gnu.org/licenses/lgpl-3.0) -->
+
+    <record id="ir_rule_res_partner_allowed_operating_units" model="ir.rule">
+        <field name="model_id" ref="base.model_res_partner" />
+        <field
+            name="domain_force"
+        >[('operating_unit_ids', 'in', operating_unit_ids)]</field>
+        <field name="name">Partner from allowed operating units</field>
+        <field name="global" eval="True" />
+        <field name="perm_read" eval="True" />
+        <field name="perm_write" eval="False" />
+        <field name="perm_create" eval="False" />
+        <field name="perm_unlink" eval="False" />
```

Comment:
the record rule should apply for the 4 operations, not only read.

Reply:
what do you mean? don't we allow only to read?